examples/article-parallax.amp.html

```diff
@@ -6,7 +6,19 @@
   <link rel="canonical" href="amps.html">
   <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
   <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <script async custom-element="amp-fx-parallax" src="https://cdn.ampproject.org/v0/amp-fx-parallax-0.1.js"></script>
```

Comment:
Do you need `custom-element` if this extension does not register an `<amp-fx-parallax>` element?